clients/python-wrapper/setup.py

```diff
@@ -15,6 +15,7 @@
     "setuptools == 68.2.2",
     "lakefs-sdk ~= 1.0",
     "pyyaml ~= 6.0.1",
+    "urllib3 ~= 2.0.7"
```

Comment:
Can we think the one we use in the underlaying python SDK? `urllib3 >= 1.25.3, < 2.1.0`? or there is specific requirement to use this one?

Reply:
We are already using this lib regardless of the SDK. I don't think we should use an older version of it just so we align ourselves with the SDK

Reply:
Don't think it is older - "urllib3 ~= 2.0.7" means 2.0.x
The lakefs sdk can go up to 2.1.0 (not included)

Reply:
Databricks cluster and many other libraries e.g. Dagster are dependent on urllib3 v1.26.x and won't work with v2.0.7. So, we should not use `urllib3 ~= 2.0.7`. I think we should go with `urllib3 >= 1.25.3, < 2.1.0` that @nopcoder suggested.

Reply:
@kesarwam this has nothing to do with databricks dependencies. When installing a package all the required dependencies should be installed for the package to work. We should not align ourself with databricks, Jupyter or any other external tool for that matter

Reply:
At the end there will be a single package loaded into the runtime. So it depends on other packages loaded your python env.